app/_hub/kong-inc/jwt-signer/_2.2.x/_index.md

```diff
@@ -14,9 +14,7 @@ can be any valid verifiable tokens. E.g. two access tokens
 * [Plugin Configuration Parameters](#plugin-configuration-parameters)
   * [Description of Plugin Configuration Parameters](#description-of-plugin-configuration-parameters)
     * [config.realm](#configrealm)
```

Comment:
I think we can safely remove the whole `app/_hub/kong-inc/jwt-signer/_2.2.x/` folder